cmake/caches/Runtime-Android-armv7.cmake

```diff
@@ -0,0 +1,28 @@
+
+set(SWIFT_HOST_VARIANT_SDK ANDROID CACHE STRING "")
+set(SWIFT_HOST_VARIANT_ARCH armv7 CACHE STRING "")
+
+# NOTE(compnerd) disable the tools, we are trying to build just the standard
+# library.
+set(SWIFT_INCLUDE_TOOLS NO CACHE BOOL "")
+
+# NOTE(compnerd) cannot build tests since the tests require the toolchain
+set(SWIFT_INCLUDE_TESTS NO CACHE BOOL "")
+
+# NOTE(compnerd) cannot build docs since that requires perl
+set(SWIFT_INCLUDE_DOCS NO CACHE BOOL "")
+
+# NOTE(compnerd) these are part of the toolchain, not the runtime.
+set(SWIFT_BUILD_SYNTAXPARSERLIB NO CACHE BOOL "")
+set(SWIFT_BUILD_SOURCEKIT NO CACHE BOOL "")
+
+# NOTE(compnerd) build with the compiler specified, not a just built compiler.
+set(SWIFT_BUILD_RUNTIME_WITH_HOST_COMPILER YES CACHE BOOL "")
+
+set(SWIFT_SDK_ANDROID_ARCHITECTURES armv7 CACHE STRING "")
+
+# NOTE(compnerd) this is lollipop, which seems to still have decent usage.
+set(SWIFT_ANDROID_API_LEVEL 21 CACHE STRING "")
+# NOTE(compnerd) this matches the value from the NDK r24.
+set(SWIFT_ANDROID_NDK_CLANG_VERSION 14.0.1 CACHE STRING "" FORCE)
```

Comment:
The latest LTS NDK 23c was just released and it uses 12.0.9, I think it'd be better to use that but up to you. I was planning on submitting a pull [updating that in the default CMake configuration](https://github.com/buttaface/swift-android-sdk/commit/dd896b481ad5a65aade46bc243d72d34c4830b5d#diff-1cc443d876544596448190488e09691d83f486fabf85e5a8b75def4fcfb05cf1R10) and the docs.

Reply:
I think that going with the latest release is reasonable, but so is the LTS release.  I think that I am going to go with this for the time being, but this is something to revisit in the future.  A more interesting thing would be to see if we can completely remove the need to specify the version and compute that dynamically instead.